com.unity.cinemachine/Editor/Upgrader/UpgradeObjectToCm3.cs

```diff
@@ -182,6 +189,8 @@ static int FindRig(string path)
         /// <param name="go">The GameObject being upgraded</param>
         public void DeleteObsoleteComponents(GameObject go)
         {
+            if (go == null) return;
```

Comment:
I think I was testing without deleting any components and I got the error here because go was null. Not something that should happen in a real situtation. It does not hurt having it here, but I can remove it.

Reply:
It doesn't hurt, but it can hide potential bugs.  It's part of general code cleanliness: better to crash early on an unexpected null than to try to find the source of a covered-up problem